src/Sheet/README.md

```diff
@@ -63,3 +63,7 @@ notes: |
   )
 }
 ```
+
+<guide
+  selectors="`pgn__sheet-component`"
```

Comment:
Looks like there is a `role="presentation"` in this component. Should it be included in the `defaultText` here?